templates/zerver/api/incoming-webhooks-overview.md

```diff
@@ -8,8 +8,7 @@ Zulip:
   messages](/api/send-message).  This works great for internal tools
   or cases where the third-party tool wants to control the formatting
   of the messages in Zulip.
-* Use one of our supported [integration
-  frameworks](/integrations/doc/meta-integration), such as the
+* Use one of our supported integration frameworks, such as the
```

Comment:
Why remove this link -- was it broken?  I think it was meant to go to that category on /integrations.

Reply:
It does, just without the /doc/ part of the URL; fixed the link before merging.